### PR TITLE
Add recent Perl distributions to registry

### PR DESCRIPTION
**Pull request by jjatria**
This commit also includes an update to the Perl API entry, that incorrectly reported its license as Apache.

I made some efforts to keep to the style of similar entries in the registry, but please let me know if this could be improved.

### CODE REVIEW

**Review by theletterf — APPROVED** [review]
Look good to me!

File: data/registry/instrumentation-perl-mojolicious.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# cSpell:ignore jjatria
```
Comment (svrnm) [inline data/registry/instrumentation-perl-mojolicious.yml:1]:
```suggestion
# cSpell:ignore jjatria Mojolicious
```

This way you can get rid of the spelling issues

**Comment by jjatria** [comment]
Oh, thank you! I was going to spend some time this evening trying to figure out how to get a recent-enough version of Node running on this machine, but you beat me to it :)

**Pull request merged**